x11-misc/stalonetray/stalonetray-0.8.4.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="System tray utility including support for KDE system tray icons"
+HOMEPAGE="https://stalonetray.github.io/"
+SRC_URI="https://github.com/kolbusa/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug +graceful-exit"
+
+RDEPEND="x11-libs/libX11
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libXpm"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto"
+BDEPEND="app-text/docbook-xml-dtd
+	dev-libs/libxslt"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf $(use_enable debug) \
+		$(use_enable graceful-exit) \
+		--enable-native-kde
```

Comment:
Can you please tell me what this does, and why apparently no KDE deps are needed?
I have a feeling it's got something to do with appindicator support? Does this then break support for older xembed system trays, ie, should this be made optional?

Reply:
Oddly enough, it's some KDE integration (I don't use KDE so can't really tell you what it does).  There was a USE flag on this, but Sam indicated to just enable it unconditionally since it doesn't actually require additional dependencies.  I'm running as per this on evilwm and can't see that anything broke ... so doesn't look like it needs to be optional.

Reply:
Looking at the code it looks like it allows stalonetray to be used as the tray manager for KDE, so instead of running the native KDE tray you can run stalonetray.  I'm afraid that's as much as I can figure out from looking at the source code.  It enables additional code, but doesn't cause *alternative* paths to be followed as far as I can tell.

Reply:
Thanks, just wanted to confirm it doesn't break for people not either on appindicator-supported or xembed-supported environments (as that's the reason to use stalonetray in the first place)